lib/SPIRV/SPIRVWriter.cpp

```diff
@@ -2972,8 +2973,8 @@ static bool tryParseAnnotationDecoValues(StringRef ValueStr,
           return false;
         // Skip the , delimiter and go directly to the start of next value.
         ValueStart = (++I) + 1;
+        continue;
```

Comment:
This changes a log of a block. I'm not sure, which would be the correct one (we may be indeed have a bug here), so please reason why it's required. Assuming that the tests are failing I'm not convinced that it's the correct change.

Reply:
In the existing logic, we identify only values enclosed within quotation marks 
Input string = "123","456","789". Output vector = {"123" "456" "789"}
it does not work for the following case:
Input string = "1,2,3". Expected output vector = {"1", "2", "3"}
With the proposed change, it works for both scenarios.
The second scenario is tested in IntelFPGALatencyControl.ll. The first scenario mostly needs a test update.
Thanks